worldedit-core/src/main/java/com/fastasyncworldedit/core/extension/factory/parser/pattern/NoXPatternParser.java

```diff
@@ -0,0 +1,46 @@
+package com.fastasyncworldedit.core.extension.factory.parser.pattern;
+
+import com.fastasyncworldedit.core.configuration.Caption;
+import com.fastasyncworldedit.core.extension.factory.parser.RichParser;
+import com.fastasyncworldedit.core.function.pattern.NoXPattern;
+import com.sk89q.worldedit.WorldEdit;
+import com.sk89q.worldedit.extension.input.InputParseException;
+import com.sk89q.worldedit.extension.input.ParserContext;
+import com.sk89q.worldedit.function.pattern.Pattern;
+import com.sk89q.worldedit.util.formatting.text.TextComponent;
+
+import javax.annotation.Nonnull;
+import java.util.stream.Stream;
+
+public class NoXPatternParser extends RichParser<Pattern> {
+
+    /**
+     * Create a new rich parser with a defined prefix for the result, e.g. {@code #simplex}.
+     *
+     * @param worldEdit the worldedit instance.
+     */
+    public NoXPatternParser(WorldEdit worldEdit) {
+        super(worldEdit, "#!x", "#nx", "#nox");
```

Comment:
Is that legacy syntax? I'm not sure if a single mask needs 3 aliases

Reply:
it's what it was in legacy, no reason not to keep them all